integrasjonspunkt/src/main/java/no/difi/meldingsutveksling/receipt/ReceiptPolling.java

```diff
@@ -28,6 +29,13 @@
     @Autowired
     private MessageReceiptRepository messageReceiptRepository;
 
+    @Autowired
+    ReceiptStrategyFactory receiptStrategyFactory;
+
+    @Autowired
+    DpiReceiptService dpiReceiptService;
+
+    // TODO: fjernes etter test!
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AS1135)